<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>n64romProperties</class>
 <widget class="QDialog" name="n64romProperties">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>152</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>294</width>
    <height>152</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>294</width>
    <height>152</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>n64 rom properties</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>cic</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cic_comboBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>cart</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6101</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6102</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6103</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6105</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6106</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cic_lineEdit">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>saveType</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="saveType_comboBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>no save/mpk</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4Kb eeprom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16Kb eeprom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32KB sram</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128KB flash</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="saveType_lineEdit">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>modeA</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="modeA_comboBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>mode0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mode15</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="modeA_lineEdit">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>n64romProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>n64romProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
